JpegXmpWritePluginMDE/JpegXmpWritePluginMDE.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Think we need to be targetting .NET 6.0 as it stands, though can multi-target 6 and 8 as the next MDE version has an 8.0 target with more optimizations etc

Reply:
I found that out why integrating the plugin in our project. I need to push this change <TargetFrameworks>net6.0;net8.0</TargetFrameworks>